service/src/config/serverConfig.ts

```diff
@@ -40,27 +40,27 @@ export const serverConfig: ServerConfig = {
       operation: [
         {
           name: 'package',
```

Comment:
Same here?

Reply:
Tried making the change here and it made all the unit tests fail. I forgot that the [asymmetrik docs] require that the name matches up with the corresponding function name, where the function name is camel case. Right now the name is `"package"` and the corresponding function is `package()`. If we were to change the name, we would need to change the function to `cqfmPackage()`. It's not a big deal if we were to add that change, but I think it's fine to keep this as just `package` for now since there is only one package function defined within the measure repository spec anyway.

Reply:
Ohhh oops! Sorry about that, I totally forgot about the asymmetrik stuff. No need to change!